packages/kbn-dev-utils/src/kbn_client/kbn_client.ts

```diff
@@ -43,10 +43,10 @@ export class KbnClient {
    */
   constructor(
     private readonly log: ToolingLog,
-    private readonly kibanaUrls: string[],
+    private readonly kibanaConfig: KibanaConfig,
```

Comment:
Why could it be an array? I switched to an object instead

Reply:
There was definitely an intention to support testing multiple Kibana instances talking to the same ES cluster when we were talking a lot about Kibana "clustering" and support for services and testing those services which carried state across a collection of Kibana instances... Which I don't think we ever turned out to really need.